docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Rotary.RotaryEncoder.md

```diff
@@ -51,23 +51,13 @@ public override Task Initialize()
     //==== Classic Events
     rotaryEncoder.Rotated += RotaryEncoder_Rotated;
 
-    //==== IObservable
```

Comment:
Did we intend to remove the Observable example here and in the next example?

Reply:
yes - and thx for checking